.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -1,20 +1,15 @@
 ---
 name: Documentation 📝
-about: Suggest better docs coverage for a particular tool or process.
+about: Suggest specific improvements for our documentation. For questions, please use "Discussions".
 labels: "type: documentation"
 ---
 
 <!--
   To make it easier for us to help you, please include as much useful information as possible.
 
   Useful Links:
-  - Documentation: https://www.gatsbyjs.org/docs/
-  - Contributing: https://www.gatsbyjs.org/contributing/
-
-  Gatsby has several community support channels, try asking your question on:
-
-  - Discord: https://gatsby.dev/discord
-  - Twitter: https://twitter.com/gatsbyjs
```

Comment:
Do you remove this intentionally? So, do we always suggest using discussions instead?

Reply:
It was outdated and also not in the bug_report template anymore, so this just aligns things and moves to singular place (the initial "config.yml" view)